osgp/platform/osgp-adapter-kafka-distributionautomation/src/main/java/org/opensmartgridplatform/adapter/kafka/da/application/config/KafkaProperties.java

```diff
@@ -0,0 +1,86 @@
+package org.opensmartgridplatform.adapter.kafka.da.application.config;
+
+import java.util.HashMap;
+import java.util.Map;
+
+// TODO - Move to a shared location for reuse
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3AS1135)

Reply:
Moved